{"id":"0d364400","title":"集群发生未知故障","date":"2021-01-06T01:45:00.000Z","modified":"2021-01-06T10:34:53.926Z","scheduled":null,"severity":"major-outage","affectedsystems":["backend","gateway","seaweedfs","judge"],"resolved":true,"content":"<p>我们接到主站无法打开的反馈，并正在调查此问题。</p>\n<div class=\"update-block\">\n          <div class=\"update-block-date\">2021-01-06T09:26:54.096Z</div>\n          <div class=\"update-block-content\">\n            <div class=\"update-block-title\">Resolved</div>\n            <div class=\"update-block-body\">\n<p>评测系统已经恢复，并重评了积压的提交。</p>\n</div></div></div>\n<div class=\"update-block\">\n          <div class=\"update-block-date\">2021-01-06T08:35:06.516Z</div>\n          <div class=\"update-block-content\">\n            <div class=\"update-block-title\">Monitoring</div>\n            <div class=\"update-block-body\">\n<p>主站服务已经恢复，正在尝试对评测系统部署修复方案。</p>\n</div></div></div>\n<div class=\"update-block\">\n          <div class=\"update-block-date\">2021-01-06T03:24:05.708Z</div>\n          <div class=\"update-block-content\">\n            <div class=\"update-block-title\">Investigating</div>\n            <div class=\"update-block-body\">\n<p>我们已经定位到问题，正在尝试部署修复方案。</p>\n</div></div></div>\n"}